src/components/Checkbox/Checkbox.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <label class="checkbox" 
+    :class="[{'checkbox__input--disabled': disabled},
+            {'checkbox__input--same-line': sameLine}]"
+  >
+    <input
+      :class="['checkbox__input', {'checkbox__input--error': error}]"
+      type="checkbox"
+      v-model="computedVModel"
+      :value="nativeValue"
+      :name="name"
+      :disabled="disabled"
+      :required="required"
+      :true-value="trueValue"
+      :false-value="falseValue"
+    />
+    <span class="checkmark"></span>
+    <label for="name" class="checkbox__label">{{label}}<template v-if="required">*</template></label>
```

Comment:
There are two `label` tags in this component - can we remove one of them? (I'd personally prefer if we removed the outer one so that the pattern of the checkbox matches that of the radio button.)

Reply:
Also, if we're structuring this this way--

```
<input id="something" />
<label for="something" />
```

The input has to have an `id` that's the same as the `for` attribute in the label. That's how a screen reader knows what label goes to what input. ☺️

Reply:
Yes, not sure why when changing the outer <label> to <div>, the click stops working... will solve that and push again